src/server/lib/validation-error.js

```diff
@@ -1,6 +1,6 @@
 
 class ValidationError extends Error {
```

Comment:
```suggestion
/**
 * severity: "err" | "warn"
 */
class ValidationError extends Error {
```

Could you add a short comment or something expressing which severities are expected?